src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/KontantstøtteValg.tsx

```diff
@@ -115,13 +129,26 @@ const KontantstøtteValg: React.FC<Props> = ({
             <Heading spacing size="small" level="5">
                 Kontantstøtte
             </Heading>
-            <JaNeiRadioGruppe
-                error={valideringsfeil?.harKontantstøtte}
-                legend={radioGruppeTekst}
-                lesevisning={erLesevisning}
-                onChange={(event) => kontantstøtte.onChange(event)}
-                value={kontantstøtte.value as ERadioValg}
-            />
+            <AlertOgRadioknappWrapper>
+                {harKontantstøtteUtbetalinger && (
+                    <AlertStripe variant={'warning'} size={'small'}>
+                        Bruker har eller har fått kontantstøtte.
+                    </AlertStripe>
+                )}
+                {!harKontantstøtteUtbetalinger && (
+                    <AlertStripe variant={'info'} size={'small'}>
+                        Bruker verken mottar eller har mottatt kontantstøtte.
+                    </AlertStripe>
```

Comment:
I "historisk" visning - så henter vi "har kontantstøtte" på nytt og viser frem en alertstripe dersom brukeren har det på visningstidspunktet - gir det mening? Eller bør vi unngå å vise en slik alert dersom en behandling er ferdigstilt?

Reply:
Hvis vi skal vise noe "historisk" så må vi isåfall lagre ned verdien hos oss basert på behandlingId-en når vi iverksetter?

Reply:
Trenger man å vise alerten i historisk visning? Kan det holde at saksbehandler har tatt et valg for dette viser om bruker har/ikke har kontantstøtte

Reply:
Fikset, alertstripe vises kun dersom det ikke er lesevisning